liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -314,6 +314,9 @@ public ServerTask getServerTask() throws Exception {
                 // Setup server task
                 serverTask = initializeJava();
                 copyConfigFiles();
+                if (project.getProperties().containsKey("container")) {
```

Comment:
Can this just use `if (container) {`? Since the container boolean is in DevMojo

Reply:
yes, it should be